data/World/Overworld.json

```diff
@@ -378,6 +421,9 @@
         "locations": {
             "Pierre": "is_adult and Bonooru and not free_scarecrow",
             "LH Underwater Item": "is_child and can_dive",
+            "LH Underwater Near Shore Green Rupee": "is_child",
+            "LH Underwater Green Rupee 1": "is_child and can_dive",
+            "LH Underwater Green Rupee 2": "is_child and can_dive",
```

Comment:
You can stand on those pillars by the LH->Dom dive warp and Boomerang the rupees from there. Obviously that would have to be a trick at best because of how weird of a place that is to stand.

However, it's possible that standing there might be a seam walk, and if so, it wouldn't be allowed. I will try to get confirmation on whether or not this trick is even allowed since people are going to want to do it regardless of whether we make a trick for it.